assemblies/assembly-planner-java.adoc

```diff
@@ -0,0 +1,750 @@
+. Follow the instrucions in the README file to run the application.
+////
+
+== Create the Maven or Gradle build file and add dependencies
+You can use Maven or Gradle for the OptaPlanner school timetable application. After you create the build files, add the following dependencies:
+
+
+* `optaplanner-core` (compile scope) to solve the school timetable problem
+* `optaplanner-test` (test scope) to JUnit test the school timetabling constraints
+* An implementation such as `logback-classic` (runtime scope) to see what OptaPlanner is doing
+
+.Procedure
+. Create the Maven or Gradle build file.
+. Add `optaplanner-core`, `optaplanner-test`, and `logback-classic` dependencies to your build file:
+
+** For Maven, add the following dependencies to the `pom.xml` file:
++
+[source]
+----
+  <dependency>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-core</artifactId>
+  </dependency>
+
+  <dependency>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-test</artifactId>
+    <scope>test</scope>
+  </dependency>
+
+  <dependency>
+    <groupId>ch.qos.logback</groupId>
+    <artifactId>logback-classic</artifactId>
+    <version>1.2.3</version>
+  </dependency>
+----
++
+The following example shows the complete `pom.xml` file.
++
+[source,xml,subs=attributes+]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>optaplanner-hello-world-school-timetabling-quickstart</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <maven.compiler.release>{java-version}</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>{project-version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>1.2.3</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+
+    <!-- Testing -->
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
```

Comment:
Need  to add compiler plugin. The application will fail to compile Java 11 syntax without it
  <plugin>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.1</version>
  </plugin>

Reply:
Add but really just guessing at the syntax so please check

Reply:
Could you tell me where did you get the initial code snippet? 
If that needs to be shown here then you can get it from this file
https://github.com/kiegroup/optaplanner-quickstarts/blob/5a913d2af38f09a128062b15c807e8a027e3bd7a/hello-world/pom.xml#L14

Reply:
It probably came from Geoffrey. But I put in your example.

Reply:
<version.org.optaplanner>8.12.0-SNAPSHOT</version.org.optaplanner> should be replaced with
<version.org.optaplanner>{optaplanner-version-product}</version.org.optaplanner>

Reply:
Updated

Reply:
LGTM